common/persistence/sql/sqlplugin/sqlite/visibility.go

```diff
@@ -298,7 +298,11 @@ func (mdb *db) processRowFromDB(row *sqlplugin.VisibilityRow) {
 	row.StartTime = mdb.converter.FromSQLiteDateTime(row.StartTime)
 	row.ExecutionTime = mdb.converter.FromSQLiteDateTime(row.ExecutionTime)
 	if row.CloseTime != nil {
-		closeTime := mdb.converter.FromSQLiteDateTime(*row.CloseTime)
-		row.CloseTime = &closeTime
+		if row.CloseTime.Equal(maxSQLiteDateTime) {
```

Comment:
why change close to NOT NULL and make max value as its default? What is the problem of allow NULL?

Reply:
It makes easier for `ORDER BY`, and pagination filters.
Otherwise, we'd have to do something like this in the queries: `COALESCE(close_time, '9999-12-31 23:59:59')`.

Reply:
Discussed with @alexshtin, and decided to keep NULL.